extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -331,6 +331,60 @@
                 <directory>${project.basedir}/src/test/resources</directory>
             </testResource>
         </testResources>
+
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <configuration>
+                    <shadedArtifactAttached>false</shadedArtifactAttached>
+                    <artifactSet>
+                        <includes>
+                            <include>org.apache.kyuubi:kyuubi-util-scala_${scala.binary.version}</include>
+                            <include>org.apache.kyuubi:kyuubi-util</include>
+                            <include>org.apache.ranger:ranger-plugins-common</include>
+                            <include>org.apache.ranger:ranger-plugins-audit</include>
+                            <include>org.codehaus.jackson:jackson-jaxrs</include>
```

Comment:
it's not a good practice to create a shaded jar that does not perform relocation

Reply:
have you checked the output shaded jar, to make sure this is no class leak?

Reply:
I decompiled check shade jar and expected that class existed. Besides, I tested shade jar manually and it worked as expected.

Reply:
Now, it only leaks `jna` related classes, and since jna contains so, it seems that it cannot be relocated.

Also, ranger-related classes do not require relocation, which does not change how the user is released.

I think this type of package has almost no class conflict issues, so it seems that this is ok too.